### ISSUE TITLE
Create function to allow a new definitions file to be added

### ISSUE THREAD

**Issue by ChrisMarsh82** [IDEMSInternational/epicsawrap#49]
Function so that a new definitions file can be added for a station.

inputs: country, station_id, definitions

This should not delete the existing definitions file but date stamp the files added. 

Only the latest definitions file should be accessed by the functions. The historical definitions will not need to be accessed at the moment as this work is out of scope of this issue.

**Comment by lilyclements** [comment]
This function belongs in `epicsadata` since it relates to the data. I have made this function under `add_definitions_to_bucket`. 
I have also amended the function to get the data from the bucket (`get_definitions_data`). This now will get the most recent definitions data if there is multiple, or otherwise will just get one. 

The PR for this in the `epicsadata` package is given here https://github.com/IDEMSInternational/epicsadata/pull/11

I've then written a wrapper function for `epicsawrap` so that this can be accessed from this package - this is in PR #57. If you're happy with this, then this issue can be closed as completed
